### ISSUE TITLE
Bug with incorrect behaviour with linkmap

### ISSUE THREAD

**Issue by hadson19** [orientechnologies/orientdb#6677]
After insert new Record with incorrect link map - our class become "locked" on UI part and it seems that we laso cannot make select from orientjs driver.
Example:
We have property mapLinkTest wth type maplink
WE are going to set map link via console
```
     create vertex RecordContent set recordContentMap = #85:0, recordContentGUID = "5357c9c4-f468-4c2e-98f0-a7442c8d6157", recordContent = {"717618f7-8bbf-49c7-8119-f83c6a8b1b02":#123:123}
```
     #123:123 - there is no any record with this @rid in out database.

WE got response the all ok:
 Created vertex 'RecordContent#81:2{recordContentMap:#85:0,recordContentGUID:5357c9c4-f468-4c2e-98f0-a7442c8d6157,recordContent:[1]} v1' in 0.011000 sec(s).

But after select from RecordContent:
select \* from RecordContent on UI part (also from nodejs app) - we got 
        "The command has not been executed"

```
 P.S. but console (console.sh) made response 
```

+----+-----+---------+---------+---------+---------------------------------------------------------------------------------------------------------------------------------+
|#   |@RID |@CLASS   |recordCon|recordCon|recordContent                                                                                                                    |
+----+-----+---------+---------+---------+---------------------------------------------------------------------------------------------------------------------------------+
|1   |#81:2|Record...|#85:0    |5357c9...|{717618f7-8bbf-49c7-8119-f83c6a8b1b02:#123:123}                                                                                  |
+----+-----+---------+---------+---------+---------------------------------------------------------------------------------------------------------------------------------+

**Comment by maggiolo00** [comment]
@hadson19 

do you have a script to reproduce it?

**Comment by lvca** [comment]
@hadson19 any news on this?

**Comment by maggiolo00** [comment]
hi @hadson19 

any news?

**Comment by maggiolo00** [comment]
Hi @hadson19 

Fixed, it will be available in 2.2.14
